src/main/java/redis/clients/jedis/Jedis.java

```diff
@@ -35,6 +35,17 @@
 public class Jedis extends BinaryJedis implements JedisCommands, MultiKeyCommands,
     AdvancedJedisCommands, ScriptingCommands, BasicCommands, ClusterCommands, SentinelCommands, ModuleCommands {
 
+  public static void destroy(Jedis jedis) {
```

Comment:
I don't really see the need of this function. If the connection is broken, it's almost sure that `quit` will always fail.